grpc/client/src/main/java/io/helidon/grpc/client/GrpcServiceClient.java

```diff
@@ -139,6 +136,27 @@ public String serviceName() {
         return observer.getFuture();
     }
 
+    /**
+     * Invoke a unary method that doesn't take any request parameter.
+     * @param methodName Method name
+     * @param <RespT> The type of response.
+     * @return The response.
+     */
+    public <RespT> CompletableFuture<RespT> unary(String methodName) {
+        //TODO: Check if the following is OK.
```

Comment:
Yes, passing empty `String` is no good as it will not work with a Proto service. Do we actually want to support various  different unary signatures on this class? There is proper support for this in the MP branch and a lot more signatures than you have here. If you are going to leave it you need to pass `io.helidon.grpc.core.proto.Types.Empty.getDefaultInstance()` instead of `""`.

Reply:
removed the method